Pachanga/Core/Sessions/EditSessionView.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  EditSessionView.swift
+//  Pachanga
+//
+//  Created by Javier Alaves on 17/8/23.
+//
+
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+import SwiftUI
+
+struct EditSession: View {
+    
+    let session: Session
+    
+    // session details
+    @State private var location: String = ""
+    @State private var sessionDate: Date = Date()
+    
+    // equipment
+    @State private var isBallAvailable: Bool = false
+    @State private var areLinesAvailable: Bool = false
+    
+    // available fields, stored in form for now
+    let fields = ["Club Muchavista", "Restaurante Xaloc", "Restaurante Niza", "Seis Perlas Campello"]
+    
+    var body: some View {
+        
+        Form {
+            Section ("Detalles") {
+                Picker("Lugar", selection: $location) {
+                    ForEach(fields, id: \.self) {
+                        Text($0)
+                    }
+                }
+                .onChange(of: location) { newValue in
+                    print("Location changed to \(newValue)" )
+                }
+
+                DatePicker("Fecha", selection: $sessionDate)
+                    .onChange(of: sessionDate) { newValue in
+                    }
+            }
+            Section ("Equipamiento") {
+                Toggle("Bola disponible", isOn: $isBallAvailable)
+                Toggle("Líneas disponibles", isOn: $areLinesAvailable)
+            }
+        }
+        .navigationTitle("Detalles de sesión")
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .navigationBarTrailing) {
+                
+                Button("Guardar") {
+                    let data: [AnyHashable : Any] = [
```

Comment:
i guess we could make this more efficient by only updating values that have been updated, it would make the operation cheaper and more efficient. but i guess it's fine for now